package.json

```diff
@@ -39,5 +41,18 @@
   "bugs": {
     "url": "https://github.com/dead/typeflex/issues"
   },
-  "homepage": "https://github.com/dead/typeflex#readme"
+  "homepage": "https://github.com/dead/typeflex#readme",
+  "prettier": {
+    "tabWidth": 2,
+    "printWidth": 120,
```

Comment:
can we add `"singleQuote": true` to avoid unnecessary changes? It seemed the repo stuck to single quotes before.

Reply:
What do you think of putting the Prettier configuration in a separate file? E.g. `.prettierrc`